libs/model-assessment/src/lib/ModelAssessmentDashboard/Controls/IndividualFeatureImportanceView/LargeIndividualFeatureImportanceView/LargeIndividualFeatureImportanceView.tsx

```diff
@@ -259,6 +259,7 @@ export class LargeIndividualFeatureImportanceView extends React.Component<
         selectedPointsIndexes: newSelections
       });
     }
+    return newSelections;
```

Comment:
nit: could we return [] instead of undefined so that we match the other function declarations of `toggleSelectionOfPoint = (index?: number): number[]`

Reply:
Nice catch! removed undefined